src/governance/snapshot/snapshot.graphql.service.ts

```diff
@@ -217,7 +205,12 @@ export class SnapshotGraphqlService extends GraphqlService {
         method: 'POST',
         headers: { 'Content-Type': 'application/json' },
         body: JSON.stringify({ params }),
+        retry: 3,
```

Comment:
Or even create a constant and use it everywhere?

Reply:
made a local constants